packages/features/bookings/lib/handleSeats/reschedule/rescheduleSeatedBooking.ts

```diff
@@ -39,11 +39,14 @@ const rescheduleSeatedBooking = async (
     select: {
       id: true,
       uid: true,
+      iCalUID: true,
+      userId: true,
       attendees: {
         include: {
           bookingSeat: true,
         },
       },
+      references: true,
```

Comment:
Add the iCalUID and references of the `newTimeSlotBooking` so that the proper 3rd party data is being updated.